param_templates/MOM_input.yaml

```diff
@@ -3173,6 +3239,15 @@ Global:
         units: m
         value:
             $OCN_GRID == "MISOMIP": 20.0
+    KV:
+        description: |
+            "[m2 s-1]
+            kinematic viscosity in the interior. The molecular value, ~1e-6
+            m2 s-1, may be used."
+        datatype: real
+        units: m2 s-1
+        value:
+            $OCN_GRID == "tx2_3v2": 0.0
```

Comment:
I am getting the following error for g16, t061, and t025 grids. Granted we will soon remove g16 and t061, but I think we should extend this conditionals to include all currently tested grids.